noxfile.py

```diff
@@ -58,7 +58,7 @@ def conda_tests(session):
     session.conda_install(
         "--file", "requirements-conda-test.txt", "--channel", "conda-forge"
     )
-    session.install("contexter", "--no-deps")
+    session.install("contextlib", "--no-deps")
```

Comment:
This line should be removed. There's no need to install contextlib, because it's part of the standard library.

```suggestion
```